appveyor.yml

```diff
@@ -9,7 +9,7 @@ environment:
 install:
   - ps: Install-Product node $env:nodejs_version
   - npm install
-  - npm install msnodesqlv8@0.6.12
+  - IF "%nodejs_version%" == "10" (npm install msnodesqlv8@0.8.2) ELSE (npm install msnodesqlv8@0.6.12)
```

Comment:
Shall we have docs to explain the version issue or leave that to the upstream lib?

Reply:
The problem is that v0.8 is the only one with stream support and v0.8 only supports node 10. I'm not sure we need to put anything in our docs about this, people using v8 will hopefully know their own limitations ;)